services/pihole/docker-compose.yml

```diff
@@ -24,7 +25,7 @@ services:
     # dns: 
     #   - ${DNS_SERVER}
     healthcheck:
-      test: ["CMD", "tailscale", "status"] # Check if Tailscale is running
+      test: ["CMD", "wget", "--spider", "-q", "http://127.0.0.1:8080/healthz"] # Check Tailscale has a Tailnet IP and is operational
```

Comment:
Please change the port accordingly to the above comment.

Reply:
Good suggestion, thankyou